doc/ACL/SAI-Proposal-Exact-Match.md

```diff
@@ -0,0 +1,123 @@
+SAI Exact Match API Proposal
+============================
+
+Title       | SAI Exact Match Proposal
+------------|--------------------------------
+Authors     | Midhun Somasundaran, Meta Inc
+            | Shrikrishna Khare, Meta Inc
+            | Jai Kumar, Broadcom Inc
+Status      | Draft
+Type        | Standards track
+Created     | 02/02/2023
+SAI-Version | 1.12
+---------------------------------------------
+
+# Overview #
+ACLs are generally implemented in hardware using TCAMs which have limited depth. The limited capacity of TCAM tables can become bottleneck for use cases such as ACL based forwarding. Some ASICs have capability to do exact match hash table lookup in the forwarding path. This allows matching packets based on a set of parameters similar to ACL TCAM matching and to set packet parameters or actions similar to ACL actions. This document proposes enhancements to the ACL spec to support exact match.
+
+## Exact Match Use Cases ##
+The model proposed solves two use cases
+
+- Increase ACL scale using exact match
+- Provide flexibility for EM to work in tandem with TCAM. For example, a TCAM ACL stage can set a metadata as part of its action which can be used as a part of the match key in EM ACL stage or vice versa.
+
+## Exact Match Limitations ##
```

Comment:
Would not call it as limitation. Its an Exact Match hash based HW table behavior

Reply:
updated the readme